main.py

```diff
@@ -829,6 +833,64 @@ def reportimage():
 
     return better_jsonify(**resp)
 
+@app.route("/suggest", methods=["GET"])
+@cache.cached(timeout=300, key_prefix='suggestions', query_string=True)
+def suggest(limit=10):
+    """ Return suggestions for query field autocompletion """
+    limit = request.args.get('limit', limit)
+    txt = request.args.get("q", "").strip()
+
+    suggestions = list()
+    whois_prefix = "Hver er "
+    whatis_prefix = "Hvað er "
+
+    prefix = None
+    if txt.startswith(whois_prefix):
```

Comment:
Probably should be `txt.lower().startswith(whois_prefix)` - OK to be permissive on the upper/lower case